ui/dashboards/src/context/DashboardProvider/DashboardProvider.tsx

```diff
@@ -80,8 +80,26 @@ export function DashboardProvider(props: DashboardProviderProps) {
   useEffect(() => {
     if (plugin === undefined) return;
     const spec = plugin.createInitialOptions();
-    // set default panel kind and spec for add panel editor
-    store.setState({ initialValues: { kind: defaultPanelKind, spec } });
+    // set default panel kind, spec, and queries for add panel editor
+    store.setState({
+      initialValues: {
+        kind: defaultPanelKind,
+        spec,
+        queries: [
+          {
+            kind: 'TimeSeriesQuery',
+            spec: {
+              plugin: {
+                kind: 'PrometheusTimeSeriesQuery',
+                spec: {
+                  query: '',
+                },
+              },
+            },
+          },
+        ],
```

Comment:
Should this default be configurable like `defaultPanelKind` or an empty array? Will all dashboard users always have the `PrometheusTimeSeriesQuery` kind with how plug-and-play this is with plugins?

Reply:
updated this to check `defaultPluginKinds`, similar to how it currently works with `defaultPanelKind`